README.md

```diff
@@ -0,0 +1,38 @@
+This project contains the source code of the backend of the smart city dashboard. It fetches data from different sources and serves as a relay for the [frontend](https://github.com/reedu-reengineering-education/smart-city-dashboard)
+
+### Running the project
+1. Clone the repo
+2. `cd smart-city-dashboard-backend`
+3. `cp docker-compose.example.yml docker-compose.yml`
+4. Change the env variables for the `app` services
+5. `docker-compose up -d`
+
+### Development
+1. Clone the repo
+2. `cd smart-city-dashboard-backend`
+3. `cp docker-compose.example.yml docker-compose.yml`
+4. Change the env variables for the `app` services
```

Comment:
s. oben

Reply:
Resolved in [7857af4](https://github.com/reedu-reengineering-education/smart-city-dashboard-backend/pull/5/commits/7857af4c135f4859a599b54fcc223f03c97239a9)